telegram-bot/main.go

```diff
@@ -1,16 +1,42 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
+	"math/rand"
+	"net/http"
 	"os"
+	"strconv"
 	"time"
 
+	"github.com/anaskhan96/soup"
 	tbot "github.com/go-telegram-bot-api/telegram-bot-api"
 )
 
 var bot *tbot.BotAPI
 
+var finallist string //variable to store final list of meetups
+
+//-------structs to parse json data from Meetup API
+type venue struct {
+	Venue string `json:"name"`
+}
+type group struct {
+	GroupName string `json:"name"`
+}
+type meetuplist struct {
```

Comment:
Why not put them in a separate file/package?

Reply:
Sure. That will make the code clean. So, should I move all the structs to a new file or the whole meetup fetching functionality?

Reply:
Since there is no meetup api client in go I guess this could be sent as whole in either a new package or file, whichever you find more usable :)

Reply:
I guess making a package would be better since that will increase its usability? What do you suggest?